odl/solvers/__init__.py

```diff
@@ -20,6 +20,10 @@
 
 __all__ = ()
 
+from . import advanced
```

Comment:
For the moment we have simply picked this name from this [comment](https://github.com/odlgroup/odl/issues/52#issuecomment-159325028) until we have found a better name.